alacritty.yml

```diff
@@ -336,6 +336,7 @@ key_bindings:
   - { key: Key0,     mods: Control, action: ResetFontSize                }
   - { key: Equals,   mods: Control, action: IncreaseFontSize             }
   - { key: Subtract, mods: Control, action: DecreaseFontSize             }
+  - { key: K,        mods: Control, action: ClearHistory                 }
   - { key: Home,                    chars: "\x1bOH",   mode: AppCursor   }
```

Comment:
Perhaps you're already aware of this, but `ctrl+k` often means "clear the rest of the line (after the cursor)". At least in e.g. bash and zsh with emacs bindings.

Reply:
heh, sorry for not cleaning up this branch before posting. The plan is to only add a default keybinding to the macOS config for <kbd>⌘ + k</kbd>.